src/alchemlyb/tests/parsing/test_gmx.py

```diff
@@ -117,12 +117,12 @@ def test_u_nk_with_total_energy():
     dataset = load_water_particle_with_total_energy()
 
     # Check if the sum of values on the diagonal has the correct value
-    assert_almost_equal(_diag_sum(dataset), 47611377946.58586, decimal=4)
+    assert_almost_equal(_diag_sum(dataset), 47611374980.34574, decimal=4)
```

Comment:
Why did the value change? Did we fix something?

Specifically, how did the new unit system change anything? We will need to be very clear in CHANGES that results will change. By how much?